nginx/opt/containerbuddy/nginx.json

```diff
@@ -16,5 +18,50 @@
       "poll": 7,
       "onChange": "/opt/containerbuddy/reload-nginx.sh"
     }
-  ]
+  ],
+  "telemetry": {
+    "name": "telemetry",
+    "url": "/telemetry",
```

Comment:
Should the URL and name be user-configured options? My reason for asking is how discovery would work in an environment where there may be different for each application.